src/lib/user/actions/update-my-dog-registration.ts

```diff
@@ -27,6 +27,7 @@ type Context = {
   update: MyDogRegistrationUpdate;
 };
 
+// TODO: Change updateMyDogRegistration params to actor, dogId, MyDogRegistration
```

Comment:
Create ticket

Reply:
Created https://www.notion.so/Change-updateMyDogRegistration-params-to-actor-dogId-MyDogRegistration-ce7b6370e5ea44cc823849d31ca31797